source/runner/DangerfileRunner.ts

```diff
@@ -66,7 +67,7 @@ export async function dangerJestConfig() {
       defaultPlatform: "danger-js",
     },
     moduleNameMapper: [],
-    moduleDirectories: ["node_modules"],
+    moduleDirectories: [path.join(process.cwd(), "node_modules")],
```

Comment:
I wonder if [`path.resolve()`](https://nodejs.org/api/path.html#path_path_resolve_paths) is more what we're looking for here. From the docs:

> [it] resolves a sequence of paths or path segments into an absolute path.